Automattic/jetpack
# Plans: Remove premium theme info

#### Changes proposed in this Pull Request:

* Removes premium theme mentions/info.

#### Jetpack product discussion

* Internal: p1HpG7-cbM-p2

#### Does this pull request change what data or activity we track or use?

* No.

#### Testing instructions:

* After pulling down changes, visit `/wp-admin/admin.php?page=jetpack#/my-plan` and make sure no JS errors are present and cards display as normal there.
* Review other proposed changes.
